src/components/document/student-group-view.tsx

```diff
@@ -86,8 +89,14 @@ export const StudentGroupView:React.FC<IProps> = ({ groupId, setGroupId }) => {
       </div>
     );
   };
+
+  const isStudentViewActiveTab = useUIStore().activeNavTab === "student-work";
+  const isChatPanelShown = useUIStore().showChatPanel;
```

Comment:
The usual pattern for hooks like this is to call the hook once and save it in a constant, then later refer to the constant. So `const ui = useUIStore(); const isChatPanelShown = ui.showChatPanel; ...`. See lines 44 & 45 above, which do something similar with `useUserStore` and `useGroupsStore`.

Reply:
Got it, will change